ast/functions.go

```diff
@@ -176,7 +176,12 @@ const (
 	Year             = "year"
 	YearWeek         = "yearweek"
 	LastDay          = "last_day"
-	TiDBParseTso     = "tidb_parse_tso"
+	// TSO functions
+	// TiDBBoundStaleness is used to determine the TS for a read only request.
+	// It will be used in the Stale Read feature.
+	// For more info, please see AsOfClause.
+	TiDBBoundStaleness = "tidb_bound_staleness"
```

Comment:
It should be `bounded staleness` rather than `bound staleness`.

Reply:
I think it's ok to use the passive voice `bound` here as an adjective, and it's shorter than `bounded`.

Reply:
I think it's ok to use the passive voice `bound` here as an adjective, and it's shorter than `bounded`.

Reply:
Fair enough. So what about `tidb_bounded_staleness` or `tidb_staleness_bound`?

Reply:
https://cloud.google.com/spanner/docs/timestamp-bounds#bounded_staleness

Reply:
I rename it to `tidb_staleness_bound`, PTAL!

Reply:
@morgo  PTAL, what do you think?

Reply:
I prefer `tidb_bounded_staleness ` . :)

Reply:
Yeah, I agree with @nolouch :-)

Reply:
@morgo @nolouch @djshow832 I finished the renaming, PTAL.